Subsystems/Electronics/Testing/ElectronicLoad/Software/src/CISBoard/PinNames.h

```diff
@@ -171,14 +171,38 @@ typedef enum {
   // Not connected
   NC = (int)0xFFFFFFFF,
 
-  STDIO_UART_TX = NC,
-  STDIO_UART_RX = NC,
+  STDIO_UART_TX = PA_9,
+  STDIO_UART_RX = PA_10,
   USBTX         = STDIO_UART_TX,
   USBRX         = STDIO_UART_RX,
 
   /*** Comms HW V1.0 ***/
   LED1 = PD_2,
 
+  /*** Electronic Load ***/
+  INPUT_CURRENT = PC_0, //ADC
```

Comment:
It is better to name your variables and functions to not need a comment right after to clarify their use: ADC_INPUT_CURRENT, ADC_INPUT_VOLTAGE. The current INPUT_CURRENT makes sense to me and doesn't need the comment ADC after but if you want it then add it to the name